### DISCUSSION TITLE
Emby Transcoding LXC in R730 (2xXeon Core, 1 GPU AMD) Proxmox 8.2.4

### DISCUSSION THREAD

**Discussion by Crosus97** [tteck/Proxmox#3491] [Help Needed]
I need help connection the GPU to Emby LXC Container, i find some guides but, is for Intel Gpus or nvidia, but outdated

TY anticipated

Emby Premiere Subscription Active ✔

**Comment by Crosus97** [comment]
Anyone know how to add a GPU or Something to the LXC container?

**Comment by gravcyber** [comment]
I am using Plex, but maybe thats gonna help you. With this config i am using the igpu inside my Ryzen 7 5700G. Its important that proxmox loads the driver for your gpu, you cant get gpu-passthrough to a vm and at the same time use the gpu in a lxc container.
![image](https://github.com/user-attachments/assets/5c084f2c-ec25-48c8-876e-33cbb155de5e)
Please check the device, you wanna connect to your lxc, i am not using an R730 Server :-)

than you can check with the following command inside your lxc container if the device gets through
ls -l /dev/dri

**Reply by gravcyber** [reply]
maybe this guide can help you, i know its for plex - but emby and plex are doing the same thing when it comes to playback and transcoding:
https://www.derekseaman.com/2023/04/proxmox-plex-lxc-with-alder-lake-transcoding.html
